### DISCUSSION TITLE
Why did we get roll back PR's for Node.js to 16.15.0?

### DISCUSSION THREAD

**Discussion by mrsimo** [renovatebot/renovate#16410] [Request Help] [answered]
On July 1st, starting ~6:30AM UTC, renovate opened roll back PR's for Node.js to go from 16.15.1 to 16.15.0 on all the repos we had Node.js using that version.

From the logs:

```json
{
  "warnings": [],
  "currentVersion": "16.15.1",
  "versioning": "node",
  "depIndex": 58,
  "depName": "node",
  "datasource": "github-tags",
  "fixedVersion": "16.15.1",
  "depType": "volta",
  "currentValue": "16.15.1",
  "updates": [
    {
      "newMajor": 16,
      "newValue": "16.15.0",
      "newVersion": "v16.15.0",
      "updateType": "rollback",
      "branchName": "renovate-node-rollback",
      "bucket": "rollback"
    }
  ],
  "packageName": "nodejs/node",
  "commitMessageTopic": "Node.js",
  "sourceUrl": "https://github.com/nodejs/node",
  "prettyDepType": "volta"
}
```

I'm having a hard time understanding the reason. From `datasource=github-tags` I understand it uses github tags to figure out versions, and that the repository it uses is `https://github.com/nodejs/node`. There [does seem to be a tag with `v16.15.1`](https://github.com/nodejs/node/releases/tag/v16.15.1) in that repository. Also:

```
❯ git ls-remote https://github.com/nodejs/node | grep tags | grep v16.15.1 | head -1
5d3fa1e7c180c2141dcfad74a3aad9f370cca305        refs/tags/v16.15.1
```

There is a `Versions found before rolling back` log entry as well, with a `versions` attribute that is listing all the versions of node it seems to have found. It's a massive array that indeed, does **not** include `v16.15.1`.

This started happening without any configuration change on our side. Is there anything obvious we're missing?

**Comment by mrsimo** [comment]
I don't know if this is life's humor, but literally on the next run of renovate after opening this discussion it autoclosed all the PR's. This was very confusing. Any pointers as to what could have happened would be welcome.

**Comment by HonkingGoose** [comment, marked as answer]
Yeah, there was something weird going on. This issue has more information:

- https://github.com/renovatebot/renovate/issues/16374

**Reply by mrsimo** [reply]
Thank you 🙏🏻 not sure how I managed to miss that issue with all the searching I did this morning.
